tests/fixtures/issues/issue-900.php

```diff
@@ -15848,112 +15848,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-900-LC' => [
-        'ua' => 'Mozilla/6.0 (Macintosh; I; Intel Mac OS X 11_7_9; de-LI; rv:1.9b4) Gecko/2012010317 Firefox/10.0a4',
-        'properties' => [
-            'Comment' => 'Firefox 10.0',
-            'Browser' => 'Firefox',
-            'Browser_Type' => 'Browser',
-            'Browser_Bits' => '32',
-            'Browser_Maker' => 'Mozilla Foundation',
-            'Browser_Modus' => 'unknown',
-            'Version' => '10.0',
-            'MajorVer' => '10',
-            'MinorVer' => '0',
-            'Platform' => 'MacOSX',
-            'Platform_Version' => '10',
-            'Platform_Description' => 'Mac OS X',
-            'Platform_Bits' => '32',
-            'Platform_Maker' => 'Apple Inc',
-            'Alpha' => false,
-            'Beta' => false,
-            'Win16' => false,
-            'Win32' => false,
-            'Win64' => false,
-            'Frames' => true,
-            'IFrames' => true,
-            'Tables' => true,
-            'Cookies' => true,
-            'BackgroundSounds' => false,
-            'JavaScript' => true,
-            'VBScript' => false,
-            'JavaApplets' => true,
-            'ActiveXControls' => false,
-            'isMobileDevice' => false,
-            'isTablet' => false,
-            'isSyndicationReader' => false,
-            'Crawler' => false,
-            'isFake' => false,
-            'isAnonymized' => false,
-            'isModified' => false,
-            'CssVersion' => '3',
-            'AolVersion' => '0',
-            'Device_Name' => 'Macintosh',
-            'Device_Maker' => 'Apple Inc',
-            'Device_Type' => 'Desktop',
-            'Device_Pointing_Method' => 'mouse',
-            'Device_Code_Name' => 'Macintosh',
-            'Device_Brand_Name' => 'Apple',
-            'RenderingEngine_Name' => 'Gecko',
-            'RenderingEngine_Version' => '10.0',
-            'RenderingEngine_Maker' => 'Mozilla Foundation',
-        ],
-        'lite' => false,
-        'standard' => true,
-    ],
-    'issue-900-LD' => [
-        'ua' => 'mozilla/3.0 (Windows NT 6.1; rv:2.0.1) Gecko/20100101 Firefox/5.0.1',
```

Comment:
Here's that rare `mozilla/3.0` prefixed UA.  I believe this is a fake.

I removed it from the suite because right now it detects as Netscape (Netscape is one of the few pattern files that has a `Mozilla/3.0` prefix, but is also probably the only legitimate one).